start.sh

```
@@ -0,0 +1,78 @@
+# Start control-plane
+PORT=${PORT:-8000}
+echo "
+[起動] Control Plane (port $PORT)..."
+cd control-plane
+./akigura-srv -listen :$PORT &
+SRV_PID=$!
+cd ..
+
+# Wait for server to be ready
+echo "[待機] サーバー起動待ち..."
+for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)